### PR TITLE
Extension 1.0.5

### PR DESCRIPTION
**Pull request by goldylucks**
TODO:
- [x] update manifests to 1.0.5 after review
- [x] merge master
- [x] build and push compiled files

### CODE REVIEW

File: extension/chrome/utils/Ws.js [hunk 1]
```diff
@@ -105,7 +105,10 @@ export default class Ws {
 
   _executeCbsWithError () {
     logger.log('[WS] executing callbacks with error: ', this._callbacks);
-    this._callbacks.forEach(cb => cb('[WS] disconnected, cb cannot be called'));
+    for (const msgId in this._callbacks) {
```
Comment (tomusdrw) [inline extension/chrome/utils/Ws.js:108]: `Object.keys(this._callbacks)` would be safier. Or `for of` loop.

File: extension/package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
     "postinstall": "npm link ../signer/web && npm link ../components",
     "start": "npm run devBuild -- -w",
     "devBuild": "webpack --progress --colors",
-    "build": "webpack --progress --colors --bail",
+    "prebuild": "npm run clean",
+    "build": "NODE_ENV=production webpack --progress --colors --bail",
```
Comment (tomusdrw) [inline extension/package.json:10]: This doesn't work on all platforms - please create a separate `build` step for yourself.
Also please remove `prebuild` - will break some caching mechanism on travis/CI.

Comment (tomusdrw) [inline extension/package.json]: This doesn't work on all platforms - please create a separate `build` step for yourself.
Also please remove `prebuild` - will break some caching mechanism on travis/CI.

Comment (goldylucks) [inline extension/package.json:10]: we have `prebuild` in all other `package.json`
should we remove them as well?

Comment (goldylucks) [inline extension/package.json:10]: `NODE_ENV=production`
how do we solve this in CI?

File: extension/chrome/utils/Ws.js (hunk 1 above)
Comment (goldylucks) [inline extension/chrome/utils/Ws.js:108]: for of doesn't work on objects
Object.keys would give the keys, and the value is what's needed here

we could do
`Object.keys(this._callbacks).forEach(msgId => this._callbacks[msgId]('cb msg here ...')`

do u have a scenario in mind where the current code wouldn't work?

File: extension/package.json (hunk 2 above)
Comment (goldylucks) [inline extension/package.json:10]: @tomusdrw 
tried [this library](https://www.npmjs.com/package/cross-env)?

Comment (tomusdrw) [inline extension/package.json:10]: Yeah, saw that - might be worth considering.

I don't want to do big changes to build system though, until we have automated full builds on windows.

Comment (goldylucks) [inline extension/package.json:10]: Anything else holding the merge?

Comment (tomusdrw) [inline extension/package.json:10]: Looks good.

**Pull request merged**